src/common/history.c

```diff
@@ -1698,20 +1698,17 @@ gboolean dt_history_paste_parts_on_list(const GList *list, gboolean undo)
   // at the time the dialog is started, some signals are sent and this in turn call
   // back dt_view_get_images_to_act_on() which free list and create a new one.
 
-  GList *l_copy = g_list_copy((GList *)list);
-
```

Comment:
This comment makes me uneasy with this changes - the list might change between time of call and time of dialog causing unexpected results?

Reply:
OK, undid that removal.  Also cleaned up the git history a bit.